appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -131,9 +130,15 @@ private GoalIdMapping createGoalIdMapping(UserDto user)
 		return GoalIdMapping.createInstance(user);
 	}
 
-	public MessageDto toMessageResource(GoalIdMapping goalIdMapping, MessageDto message)
+	public HttpEntity<MessageDto> createOkResponse(GoalIdMapping goalIdMapping, MessageDto message)
```

Comment:
As order of parameters I would expect `message` first as in `ControllerBase`.

Reply:
In ``ControllerBase``, there is no ``goalIdMapping``, so I there is no order to align with, right?